### PR TITLE
add dropdowns for tunings

### PR DESCRIPTION
**Pull request by alyssa-blair**
Add in dropdowns to be able to update the tuning. Changes intiate callback OnTuningChanged which sends the note and string that was changed to UpdateTuning, which will later need to call the backend for the updated tunings or however we want to implement it. Dropdowns are ugly rn, sounds like a later problem.

### CODE REVIEW

File: Views/FretBoard.xaml.cs [hunk 1]
```diff
@@ -9,15 +9,15 @@ namespace Fretty.Views;
...
-        {"D", [3, 2, "#FFFFD797"]}, 
-        {"A", [4, 6, "#FF75FEF6"]},
+        {"E", [0, "#FF97BFFF"]},
+        {"B", [1, "#FF97FFA3"]}, 
+        {"G", [2, "#FFFF97F3"]}, 
+        {"D", [3, "#FFFFD797"]}, 
+        {"A", [4, "#FF75FEF6"]},
     };
-
+
```
Comment (jaredwebber) [inline Views/FretBoard.xaml.cs:20]:
```suggestion

```

File: Views/FretBoard.xaml.cs [hunk 2]
```diff
@@ -27,20 +27,50 @@ public FretBoard()
         GenerateFretBoard(numRows, numCols);
         GenerateFretDots([[2, 5, 2, 1], [2, 7, 2, 1], [2, 9, 2, 1], [2, 11, 2, 1], [1, 13, 2, 1], [3, 13, 2, 1], [2, 16, 2, 2]]);
     }
-
+
```
Comment (jaredwebber) [inline Views/FretBoard.xaml.cs:30]:
```suggestion

```

File: Views/FretBoard.xaml.cs [hunk 3]
```diff
@@ -161,13 +189,14 @@ private void GenerateNote(string note, string color, int col, int row, bool bold
...
-            button.FontAttributes == FontAttributes.Bold ? FontAttributes.None : FontAttributes.Bold;
+    }
 
+    private static void UpdateTuning(string note, int guitarString)
+    {
+        // TODO: implement this function to update the string that changed
     }
-    
-}
+}
```
Comment (jaredwebber) [inline Views/FretBoard.xaml.cs:202]:
```suggestion
}

```

File: Views/FretBoard.xaml.cs [hunk 4]
```diff
@@ -27,20 +27,50 @@ public FretBoard()
...
+        picker.SelectedIndexChanged += OnTuningChanged;
+        
+        Grid.SetRow((BindableObject)picker, row);
+        Grid.SetColumn((BindableObject)picker, 0);
+        Grid.Children.Add(picker);
+    }
+
+    private static void OnTuningChanged(object? sender, EventArgs e)
+    {
+
```
Comment (jaredwebber) [inline Views/FretBoard.xaml.cs:57]:
```suggestion
```

File: Views/FretBoard.xaml.cs [hunk 5]
```diff
@@ -27,20 +27,50 @@ public FretBoard()
...
+        if (sender == null) return;
+        Picker picker = (Picker) sender;
+        string note = (string) picker.SelectedItem;
+        int guitarString = int.Parse(picker.AutomationId);
+        UpdateTuning(note, guitarString);
+
```
Comment (jaredwebber) [inline Views/FretBoard.xaml.cs:63]:
```suggestion
```

**Review by jaredwebber — CHANGES_REQUESTED** [review]
pls fix 😡

**Comment by wrighted** [comment]
> pls fix 😡

what are u doing here.

**Review by wrighted — APPROVED** [review]
Not 100% sure what this does for the use case but seems to work.

**Comment by alyssa-blair** [comment]
> pls fix 😡

Imma add more whitespace in spite of you

**Pull request merged**